cow-corpus-proc/combine_corpus_files.py

```diff
@@ -71,30 +73,69 @@ def get_sent_list(relevant_folders):
 
 if __name__ == "__main__":
     path_to_corpus = sys.argv[1] # sys.argv[1] is the first argument passed to the program through e.g. pycharm (or command line). In Pycharm, look at Running Configuration
-    print('Working with corpus {}'.format(path_to_corpus))
     target_folders = "essays"
+    folders_to_find = "annotated"  # Second argument passed to the program
+
+    '''
+    Use the function find_relevant_folders to find folders named "essays" in the corpus.
+    Print the path to each folder.
+    Print number of folders found.
+    '''
```

Comment:
Alright; I'd just say: "Sanity check report". In other words, you don't need to describe what each line is doing, rather: what you as a person what from this whole block of code (ideally, just one specific thing such as "report statistics", "sanity check of numbers in the corpus", "saving output in a file", or what not.